### PR TITLE
Update "Making your first figure" tutorial

### LINKED ISSUE #1995: Suggested change to the "Making your first figure" tutorial
In the [Set the projection](https://www.pygmt.org/dev/get-started/first_figure.html#set-the-projection) subsection, it says:

> This figure now has its colors set, but there is no projection or size set for the map.

It's not correct. When `projection` is specified, the default projection is `JQ15c+`.

### PR DESCRIPTION
**Pull request by michaelgrund**
**Description of proposed changes**

Fixes #1995 

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Comment by seisman** [comment]
xref https://github.com/GenericMappingTools/gmt/issues/6855

File: examples/get-started/first_figure.py [hunk 1]
```diff
@@ -91,8 +91,9 @@
 # Set the projection
 # ------------------
 #
-# This figure now has its colors set, but there is no projection or size
-# set for the map. Both of these values are set using the ``projection``
+# This figure now has its colors set. By default the projection and size
+# of the map is set to ``"JQ15c+``". However, both of these values can be
```
Comment (seisman) [inline examples/get-started/first_figure.py:95]:
```suggestion
# of the map is set to ``"Q15c+du``". However, both of these values can be
```

Comment (seisman) [reply examples/get-started/first_figure.py:95]: Better to explain the meaning of `Q15c+du`.

Comment (michaelgrund) [reply examples/get-started/first_figure.py]: Is it really ``+du`` by default @seisman ? Based on https://docs.generic-mapping-tools.org/dev/cookbook/map-projections.html it should be ``+dw`` shouldn't it?

File: examples/get-started/first_figure.py [hunk 2]
```diff
@@ -91,16 +91,19 @@
 # Set the projection
 # ------------------
 #
-# This figure now has its colors set, but there is no projection or size
-# set for the map. Both of these values are set using the ``projection``
-# parameter.
+# This figure now has its colors set. By default the projection and size
+# of the map is set to ``"Q15c+du"``. Here, ``"Q15c"`` defines that a
+# cylindrical equidistant map projection with a width of 15 cm is used
+# and ``"+du"`` selects the minimum map dimension. However, both of
```
Comment (seisman) [inline examples/get-started/first_figure.py:97]: > ``"Q15c"`` defines that a cylindrical equidistant map projection with a width of 15 cm is used and ``"+du"`` selects the minimum map dimension.

This sentence needs some improvements. `Q15c+du` means that the minimum map dimension (either map width or height) is 15 cm.

File: examples/get-started/first_figure.py [hunk 3]
```diff
@@ -91,16 +91,19 @@
...
+# these values can be customized according to the requirements using
+# the ``projection`` parameter.
```
Comment (seisman) [inline examples/get-started/first_figure.py:99]:
```suggestion
# of the map is set to ``"Q15c+du"``. Here, ``"Q"`` defines a
# cylindrical equidistant map projection, ``"15c+du"`` means setting
# the maximum (upper) map dimension to 15 cm. However, both of
# these values can be customized according to the requirements using
# the ``projection`` parameter.
```

**Review by seisman — APPROVED** [review]

**Pull request merged**